_posts/2015-09-xx-creating-systems-with-pipelining.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "creating systems with pipelining"
+date: "2015-09-xx"
```

Comment:
Would be cool, if we could publish this post already in August. Consulting is ok with publishing theirs in September.